Makefile

```diff
@@ -35,16 +35,16 @@ hooks: venv
 
 .PHONY: build
 build: venv  ## Compile and install Daft for development
-	@unset CONDA_PREFIX && source $(VENV_BIN)/activate && maturin develop --extras=all
```

Comment:
We could remove this `source $(VENV_BIN)/activate` as `maturin` offers some automatic support when using `.venv` instead of `venv`.